package.json

```diff
@@ -16,12 +18,16 @@
     "plugin"
   ],
   "author": "Mircea Alexandru",
```

Comment:
@AdrieanKhisbe please add yourself as contributor.
I will merge it tomorrow.

Thank you